test/test_express_cliff.jl

```diff
@@ -1,45 +1,46 @@
-using Test
-using QuantumSymbolics
-using QuantumClifford
+@testitem "Clifford" begin
+    using QuantumClifford
+    using QuantumOptics
```

Comment:
I had to add this, otherwise the tests fail. Not sure why this worked before. Apparently the "SafeTests" were not as safe as one would have hoped?

Reply:
We use package extensions for a lot of functionality between these 3 packages. Probably some weird interaction between them and the module sandboxing. Thanks for flagging this.